measure.rb

```diff
@@ -1249,6 +1274,38 @@ def self.add_walls(runner, model, building, spaces, subsurface_areas)
     return true
   end
 
+  def self.add_neighbors(runner, model, building, wall_length)
+    foundation_top = get_foundation_top(model)
+    building_construction_values = HPXML.get_building_construction_values(building_construction: building.elements["BuildingDetails/BuildingSummary/BuildingConstruction"])
+
+    shading_surfaces = []
+    building.elements.each("BuildingDetails/BuildingSummary/Site/extension/Neighbors/NeighborBuilding") do |neighbor_building|
+      neighbor_building_values = HPXML.get_neighbor_building_values(neighbor_building: neighbor_building)
+      azimuth = neighbor_building_values[:azimuth]
+      distance = neighbor_building_values[:distance]
+
+      wall_height = 8.0 * building_construction_values[:number_of_conditioned_floors_above_grade]
+      z_origin = foundation_top
```

Comment:
I suggest setting z_origin to be 0 (so that the shading surface starts at grade even for pier & beam foundations). And wall_height is currently ignoring attic surfaces -- can we use the max y-value of any model surface instead?

With these changes, you should be able to remove lines 1277-1278.